CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Linkage Checker Enforcer Rule Change Log
+
+## 1.0.1
+* The enforcer rule now works with artifacts having `${os.detect.classifier}` in their dependencies.
+  When such artifacts were resolved without [os-maven-plugin](https://github.com/trustin/os-maven-plugin)
```

Comment:
This is added explanation.

Reply:
How about simply, "The enforcer rule now recognizes the `${os.detect.classifier}` property defined by the [os-maven-plugin](https://github.com/trustin/os-maven-plugin)."

That said, I wonder if we have a much bigger issue here. The current rule is is just patching over this one special case. There are many more cases involving os-maven-plugin and potentially other plugins where we are not resolving dependencies in the same way Maven does.

Reply:
Updated the changelog. Yes, there may be other cases. So far, only os-maven-plugin. I'm thinking to handle them in an ad-hoc manner.